eZ/Bundle/EzPublishCoreBundle/Command/NormalizeImagesPathsCommand.php

```diff
@@ -68,6 +74,12 @@ protected function configure()
 
         $this
             ->setDescription('Normalizes stored paths for images.')
+            ->addOption(
+                self::SKIP_HASHING_COMMAND_PARAMETER,
+                null,
+                InputOption::VALUE_NONE,
+                'Skip filenames hashing'
```

Comment:
Even better (though that version was grammatically correct :) )
```suggestion
                'Skip filename hashing'
```